src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -278,8 +278,8 @@ def test_acr_create_build_task(self, resource_group, resource_group_location):
             'rg_loc': 'eastus',
             'sku': 'Standard',
             # This token requires 'admin:repo_hook' access. Recycle the token after recording tests.
-            'git_access_token': '934939417e99892b59660637cd4b4abeefccff04',
-            'context': 'https://github.com/djyou/BuildTest',
+            'git_access_token': 'b67dce55c2d8a654a4c823751a58aac1e59d9641',
+            'context': 'https://github.com/xiadu94/BuildTest',
```

Comment:
Does the test rely on the existences of this repo to be functional?

Reply:
No

Reply:
To record, I think it's yes.

Reply:
you should create test repository under Azure organization to park the resource. It should not rely on a personal repository.

Reply:
this test requires ownership of the repository to create the token to create the webhook , so it makes not much sense to create a test repository under azure organization.

Reply:
The concern is any future personal change will impact the test integrity. I'm fine to leave it as is for now but please make sure there is a contingency plan for the test repo.